packages/core/src/middleware/koa-management-api-hooks.ts

```diff
@@ -49,13 +49,16 @@ export const koaManagementApiHooks = <StateT, ContextT extends IRouterParamConte
     await next();
 
     // Auto append pre-registered management API hooks if any
-    const { path, method, body, status, _matchedRoute } = ctx;
+    const { path, method, body, status, _matchedRoute, params } = ctx;
     const hookRegistrationKey = buildManagementApiDataHookRegistrationKey(method, _matchedRoute);
 
-    // TODO: @simeng-li do we need to insert the request body to the hook context?
     if (hasRegisteredDataHookEvent(hookRegistrationKey)) {
       const event = managementApiHooksRegistration[hookRegistrationKey];
-      dataHooks.appendContext({ event, data: { path, method, body, status } });
+
+      dataHooks.appendContext({
+        event,
+        data: { path, method, body, status, params, _matchedRoute },
```

Comment:
it's kind of confusing of `body` here since it may refer to the request body (as `method` and `params` are for the request) better to use `response.body`?